spec/factories/publication.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :publication do
+    title    { Faker::Lorem.word }
+    content { Faker::Lorem.paragraph }
+    user
+  end
+end
```

Comment:
It seems you have a problem with line endings in your editor. :nail_care:

Reply:
I noticed after the merge, sorry.